jest.config.js

```diff
@@ -7,10 +7,8 @@ module.exports = {
   resetMocks: false,
   setupFiles: ['./jest.setup.js', 'jest-localstorage-mock'],
   setupFilesAfterEnv: ['@testing-library/jest-dom/extend-expect'],
-  globals: {
```

Comment:
Define ts-jest config under globals is now deprecated. Please define the config via transformer config instead. https://github.com/kulshekhar/ts-jest/blob/main/CHANGELOG.md#2900-2022-09-08